cli/README.md

```diff
@@ -0,0 +1,72 @@
+# Subsembly CLI
```

Comment:
I think that this Readme should be in the main directory. Users will be using the CLI to develop the runtimes so it would be better put there. Its just that the code of the actual CLI is in this folder